integrationtests/src/test/java/org/neo4j/procedure/UserFunctionIT.java

```diff
@@ -712,14 +713,26 @@ public void shouldListAllFunctions() throws Throwable
         "| 'org.neo4j.procedure.simpleArgument'                | 'org.neo4j.procedure.simpleArgument(someValue :: INTEGER?) :: (INTEGER?)'                                                                                    | ''                      |" + lineSeparator() +
         "| 'org.neo4j.procedure.squareDouble'                  | 'org.neo4j.procedure.squareDouble(someValue :: FLOAT?) :: (FLOAT?)'                                                                                          | ''                      |" + lineSeparator() +
         "| 'org.neo4j.procedure.squareLong'                    | 'org.neo4j.procedure.squareLong(someValue :: INTEGER?) :: (INTEGER?)'                                                                                        | ''                      |" + lineSeparator() +
+        "| 'org.neo4j.procedure.sum'                           | 'org.neo4j.procedure.sum(numbers :: LIST? OF NUMBER?) :: (NUMBER?)'                                                                                          | ''                      |" + lineSeparator() +
```

Comment:
@boggle this is not a built-in-function it is a normal user-defined function added in this test class.